### DISCUSSION TITLE
ECMLoss 算法支持

### DISCUSSION THREAD

**Discussion by vansinhu** [open-mmlab/OpenMMLabCamp#173] [MMDetection]
# ECMLoss 算法支持 

参考 [https://github.com/janghyuncho/ECM-Loss/blob/main/mmdet/models/losses/effective_class_margin_loss.py](https://github.com/janghyuncho/ECM-Loss/blob/main/mmdet/models/losses/effective_class_margin_loss.py)  中关于 ECMLoss 的实现，在 MMDet 3.x 中支持 ECMLoss，并在 LVIS 数据集上复现精度

## Task Information

**Repository**：MMDetection

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by zhangyiyuan1112** [comment]
Claim the mission

**Reply by zhangyiyuan1112** [reply]
https://github.com/open-mmlab/mmdetection/pull/10971
